packages/docs-app/src/examples/datetime-examples/common/precisionSelect.tsx

```diff
@@ -13,17 +13,17 @@ export interface IPrecisionSelectProps {
     /**
      * The precision-string option to display as selected.
      */
-    value: TimePrecision;
+    value: TimePrecision | "none" | undefined;
 
     /**
      * The callback to fire when the selected value changes.
      */
     onChange: (event: React.FormEvent<HTMLElement>) => void;
 
     /**
-     * Whether or not to allow an empty option.
+     * Whether or not to allow a `"none"` option.
      */
-    allowEmpty?: boolean;
+    allowNone?: boolean;
```

Comment:
Can we add a toggle for this in the `timePickerExample`? Just for completeness. Curious what it feels like in the wild.

Reply:
oh it's not supported in `TimePicker`. if you're using that element then there's no concept of "none".

Reply:
it's only for `DatePicker` where the `TimePicker` is _conditionally rendered_